addons/withWeb/data-url.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const fs = require('fs');
+const mime = require('mime');
+
+module.exports = {
+    process(src, filename) {
+        const mimeType = mime.getType(filename);
```

Comment:
Does `mime.getType` return null if it was unable to guess the mimetype or does it throw? If it returns null we should perhaps throw an error?